test/v2/models/responses/hateoas/HateoasDepartureMessageResponseSpec.scala

```diff
@@ -57,12 +55,10 @@ class HateoasDepartureMessageResponseSpec extends AnyFreeSpec with Matchers with
         "self"      -> Json.obj("href" -> s"/customs/transits/movements/departures/${departureId.value}/messages/${messageId.value}"),
         "departure" -> Json.obj("href" -> s"/customs/transits/movements/departures/${departureId.value}")
       ),
-      "departure" -> Json.obj(
-        "id" -> s"/customs/transits/movements/departures/${departureId.value}"
-      ),
-      "id"          -> s"/customs/transits/movements/departures/${departureId.value}/messages/${messageId.value}",
-      "received"    -> "2022-08-04T11:52:59",
-      "messageType" -> MessageType.DepartureDeclaration.code,
+      "id"          -> messageId.value,
+      "departureId" -> departureId.value,
+      "received"    -> "2022-08-04T11:52:59Z",
```

Comment:
I noticed you put the `Z` on the end of the value here in the test, but we've not used that elsewhere. So I did a little digging, because I thought that we specified that we used UTC everywhere... and I actually can't find that, not on our docs, nor on DevHub.

However, in both version 1 of our API, along with this upcoming version 2, we've _not_ been doing that, and I'm keen to keep things as simple as possible when it comes to developers transitioning from v1 to v2. 

I'd not be against raising a spike to decide this point, but right now, the datetime format should remain consistent with what was before -- so please drop the Z for now.

Reply:
I'm going to write a ticket about this to make this consistent across all calls. The Z shouldn't harm a ISO8601 compliant parser...